views/js/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -365,7 +365,7 @@ define([
         var $container = containerHelper.get(interaction);
         var attributes = interaction.getAttributes();
         var responseDeclaration = interaction.getResponseDeclaration();
-        var baseType = responseDeclaration.attr('baseType');
+        var baseType = responseDeclaration ? responseDeclaration.attr('baseType') : '';
```

Comment:
Default baseType to empty string is not a good idea. It may break response format validation during response processing. That said, have you noticed a case when the responseDeclaration is null or undefined ? If it is the case, then finding the source for an empty responseDeclaration is the way to go.